cw/simple_capture_aes_sca.yaml

```diff
@@ -5,9 +5,10 @@ cwfpgahusky:
   baudrate: 115200
   output_len_bytes: 16
   pll_frequency: 100000000
+  target_clk_mult: 0.1
```

Comment:
with this config, it captures 120 clock cycles (can be counted in the plot window) since the target_clk_mult does not match the actual clk divider in the FPGA. The clk frequency is actually 20 MHz instead of 10 Mhz as suggested here.

IMO correct would be to set `target_clk_mult: 0.2` and `num_cycles: 120` and `offset_cycles: -4`

Reply:
I am 100% sure it's capturing 60 target clock cycles with these settings (as before!). The AES encryption matches pretty nicely into the capture window. AES takes exactly 56 clock cycles. If we would capture 120 clock cycles, the AES trace would only span half the capture window / the other half would be "empty".

Can you please elaborate on why you think to observe 20 MHz target clock frequency? I would like to understand why we have different observations here. Maybe you want to upload the captured wave plot?

Reply:
Thanks @vogelpi, adding from our offline chat: 
- We have a 50% duty cycle clock on the target. 
- Also it seems the inactive clock edge creates a similarly significant peak as the active clock edge. 
- Hence we need to expect two significant peaks per clock period and the original settings check out :)
![image](https://github.com/lowRISC/ot-sca/assets/103024378/16da73cd-a724-497d-89ac-b0c6add229d6)